### ISSUE TITLE
Driver GPS is not being captured in the system

### ISSUE THREAD

**Issue by nitishMehrotra** [MOOVE-Network/moove-webapp#31]
**User Story Ref:**DRI.UC8.US3

**Description:**
 The GPS location of the driver needs to be captured. This is to ensure that the trip summary generated at the end of the trip contains actual moved data.
Currently the system is generating trip summary based on the the indicated route and not the travelled route.

**Comment by nitishMehrotra** [comment]
Dependent on Issue #32

**Comment by laurablamo** [comment]
I am pretty sure we declared this OUT OF SCOPE for the pilot

**Comment by yurixus** [comment]
@nitishMehrotra We had discussion about this in September in BLAMO office, the conclusion we came to was that for the trip summary we are going to take real elapsed time and for mileage: number of miles from google maps route.

We'll have to stick by this decision for the time being and implement full-on GPS tracking in Phase 2 (Post Feb 7).

**Comment by danielspils** [comment]
This was ruled out long ago. In the last 2 days has this become high priority for this coming Monday, Jan 16? Please let us know.
